src/lib/specialistBios/index.test.ts

```diff
@@ -0,0 +1,42 @@
+import gql from "lib/gql"
```

Comment:
tests go in a `__tests__` folder, as seen every else in MP

Reply:
✅